src/lib/expansion-panel/expansion-panel.test.ts

```diff
@@ -419,6 +422,153 @@ describe('Expansion Panel', () => {
+            <div id="content">Content</div>
+          </forge-expansion-panel>
+        </div>
+      `);
+      const trigger1 = el.querySelector('#button-id1') as HTMLElement;
+      const trigger2 = el.querySelector('#button-id2') as HTMLElement;
+      const expansionPanel = el.querySelector('forge-expansion-panel') as IExpansionPanelComponent;
+      const content = el.querySelector('#content') as HTMLElement;
+
+      trigger1.click();
+      expect(expansionPanel.open).to.be.true;
+
+      expansionPanel.trigger = 'button-id2';
+      expect(trigger1.getAttribute('aria-controls')).to.be.null;
+      expect(trigger1.getAttribute('aria-expanded')).to.be.null;
+      trigger1.click();
+      expect(expansionPanel.open).to.be.true;
+      expect(trigger2.getAttribute('aria-controls')).to.not.be.null;
+      expect(trigger2.getAttribute('aria-controls')).to.equal(content.getAttribute('id'));
+      expect(trigger2.getAttribute('aria-expanded')).to.equal('true');
+      trigger2.click();
+      expect(trigger2.getAttribute('aria-expanded')).to.equal('false');
+      expect(expansionPanel.open).to.be.false;
+    });
+
+    it('should update trigger aria on content change', async () => {
+      const el = await fixture<HTMLElement>(html`
+        <div>
+          <div>
+            <button id="button-id"></button>
+            <forge-expansion-panel trigger="button-id">
+              <div id="content1">Content 1</div>
+            </forge-expansion-panel>
+          </div>
+
+          <div id="content2">Content 2</div>
+        </div>
+      `);
+      const trigger = el.querySelector('#button-id') as HTMLElement;
+      const expansionPanel = el.querySelector('forge-expansion-panel') as IExpansionPanelComponent;
+      const content1 = el.querySelector('#content1') as HTMLElement;
+      const content2 = el.querySelector('#content2') as HTMLElement;
+
+      expect(trigger.getAttribute('aria-controls')).to.equal(content1.getAttribute('id'));
+      expect(trigger.getAttribute('aria-expanded')).to.equal('false');
+      content1.remove();
+      await elementUpdated(el);
+      expect(trigger.getAttribute('aria-controls')).to.be.null;
+      expect(trigger.getAttribute('aria-expanded')).to.equal('false');
+      expansionPanel.appendChild(content2);
+      await elementUpdated(el);
+      expect(trigger.getAttribute('aria-controls')).to.equal(content2.getAttribute('id'));
+      expect(trigger.getAttribute('aria-expanded')).to.equal('false');
+    });
+
+    it('should not error if no slotted content', async () => {
+      await fixture<HTMLElement>(html`
+        <div>
+          <button id="button-id"></button>
+          <forge-expansion-panel trigger="button-id"></forge-expansion-panel>
+        </div>
+      `);
+    });
+
+    it('should not error if trigger not found', async () => {
+      await fixture<IExpansionPanelComponent>(html`<forge-expansion-panel trigger="button-id"></forge-expansion-panel>`);
+    });
```

Comment:
Tests without assertions are typically bad practice (even though we know that exceptions can cause a test to fail) but most of the time these result in silent errors and false positives.